infullmvp-kotlin-basetest/src/main/java/com/infullmobile/android/infullmvp/basetest/InFullMvpCustomViewBaseTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.infullmobile.android.infullmvp.basetest
+
+import com.infullmobile.android.infullmvp.InFullMvpView
+import com.infullmobile.android.infullmvp.PresentedCustomView
+import com.infullmobile.android.infullmvp.Presenter
+import org.junit.Before
+import org.robolectric.RuntimeEnvironment
+
+abstract class InFullMvpCustomViewBaseTest<
+        T : InFullMvpView<PresenterType, PresentedViewType>,
+        PresenterType : Presenter<PresentedViewType>,
+        out PresentedViewType : PresentedCustomView<PresenterType>> {
+
+    lateinit var testedCustomView: T
+
+    val testedPresenter: PresenterType
```

Comment:
provide specific type PresenterType

Reply:
I'm getting an error when I add    ` provided project(path: ':sample-kotlin') ` to gradle. Can't find a solution